packages/devtools_app/lib/src/debugger/flutter/controls.dart

```diff
@@ -71,16 +71,34 @@ class DebuggingControls extends StatelessWidget {
                   ],
                 ),
               ),
-              const Expanded(child: SizedBox(width: denseSpacing)),
+              const SizedBox(width: denseSpacing),
               RoundedOutlinedBorder(
                 child: Center(
                   child: Padding(
                     padding: const EdgeInsets.only(
-                        left: denseSpacing, right: borderPadding),
+                        left: defaultSpacing, right: borderPadding),
                     child: BreakOnExceptionsControl(controller: controller),
                   ),
                 ),
               ),
+              const Expanded(child: SizedBox(width: denseSpacing)),
+              ValueListenableBuilder(
+                valueListenable: controller.librariesVisible,
+                builder: (BuildContext context, value, Widget child) {
```

Comment:
rename value to `visible` and then below, use `visible` instead of `controller.librariesVisible.value`

Reply:
👍 done